modules/zcbor/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+if(CONFIG_ZCBOR)
+  add_library(ZCBOR INTERFACE)
+
+  zephyr_include_directories(
+    ${ZEPHYR_ZCBOR_MODULE_DIR}/include
+    ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I think this line is not needed (I don't see any header files in this directory)

Reply:
Fixed